superset/utils/core.py

```diff
@@ -1481,3 +1498,7 @@ def get_time_filter_status(  # pylint: disable=too-many-branches
 def format_list(items: Sequence[str], sep: str = ", ", quote: str = '"') -> str:
     quote_escaped = "\\" + quote
     return sep.join(f"{quote}{x.replace(quote, quote_escaped)}{quote}" for x in items)
+
+
+def find_duplicates(items: Iterable[Any]) -> List[Any]:
+    return [item for item, count in collections.Counter(items).items() if count > 1]
```

Comment:
`Any` can be made [generic](https://mypy.readthedocs.io/en/latest/generics.html#generic-functions) here:
```suggestion
from typing import TypeVar

T = TypeVar('T')


def find_duplicates(items: Iterable[T]) -> List[T]:
    return [item for item, count in collections.Counter(items).items() if count > 1]
```

Reply:
Updated.